musicbrainz/csv/README.md

```diff
@@ -0,0 +1,10 @@
+# MusicBrainz JSON -> CSV
+Steps:
+1.  Get the latest JSON files from https://data.metabrainz.org/pub/musicbrainz/data/json-dumps
+    *  Examples in data folder.
+2.  Run convert_to_csv.py, specify the entity type in the second argument.
+    *  Example command line: 
+            cd musicbrainz/csv
+            python3 convert_to_csv.py data/test_area area
+3.  A out.csv should be generated. Rename it and save it.
```

Comment:
If I have to rename this file every time, why is this not a command line argument to the script?

Reply:
I changed the code but forgot to change it here. The output name should follow the input entity_type.